app/views/docs/query.phtml

```diff
@@ -0,0 +1,276 @@
+<p>
+Many Appwrite endpoints allows you to filter returned results through queries. Appwrite's SDKs include a <span class="tag">Query</span> class that helps construct query strings. This page documents all supported operations for Client SDKs. Server SDKs will behave similarly. 
+</p>
+
+<h2><a href="/docs/query#equal" id="equal">Equal</a></h2>
+<p>
+Returns results where the specified <b>attribute</b> is exactly equal to <b>value</b>.
```

Comment:
In all these examples , we should mention that value can either be a single value like `Avengers` or a list of values like `['Avengers', 'Titanic', 'Iron Man']`

Reference : https://github.com/appwrite/sdk-for-web/blob/master/src/sdk.ts#L3748-L3767